journalbeat/_meta/fields.common.yml

```diff
@@ -0,0 +1,299 @@
+- key: common
+  title: "Common Journalbeat"
+  description: >
+    Contains common fields available in all event types.
+  fields:
+    - name: coredump
```

Comment:
Do you expect all these fields like `coredump`, `kernel` to be on the top level?

Reply:
Yes, because I specify the field names here: https://github.com/elastic/beats/pull/8277/files#diff-e6feda99a2ce30c6c943d4a15664abd4R24

I am OK with moving them to lower levels to fit better into ECS. I haven't found any namespace which could contain these fields. Do you have any suggestions?

Reply:
Are they all related to journald? If yes, I wonder if we should prefix them with journald?

Reply:
It would be nice to have some example json events in the future somewhere which we can use for the docs but should also make it much clearer how the events will look like.

Reply:
Yes, these are all related to journald. Here is the full list of fields provided by systemd journal: https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html
But some of the fields are not coming from the journal, but an application appends some fields. For example `coredump` is provided by `systemd-coredump` kernel helper. So prefixing with `journald` might be confusing.

Reply:
Are all the fields part of systemd? In general I would suggest to first prefix them with "something" and make them more global over time. At the same time not a blocker.

Reply:
Ok. I will prefix fields which does not belong anywhere in ECS with `journald` in a follow up.

Reply:
Ok, lets have a quick chat before you put all the work into prefixing it.

Reply:
It's too late. :D https://github.com/elastic/beats/pull/8324/commits/80aa2e4e00ef1b4834bf62b9496a1e473c961409